### PR TITLE
Update the isRemoteXact flag into a independent bool

### PR DESCRIPTION
**Pull request by poojanilangekar**
Same as #36

### CODE REVIEW

File: src/backend/storage/lmgr/proc.c [hunk 1]
```diff
@@ -440,6 +440,7 @@ InitProcess(void)
 	Assert(pg_atomic_read_u32(&MyProc->clogGroupNext) == INVALID_PGPROCNO);
 
 	pg_atomic_init_u64(&MyProc->assignedXidCsn, InProgressXidCSN);
```
Comment (ctring) [inline src/backend/storage/lmgr/proc.c:442]: tag here

File: src/include/storage/proc.h [hunk 2]
```diff
@@ -308,6 +307,18 @@ struct PGPROC
 	 */
 	CSN_atomic assignedXidCsn;
 
+	/*
+	 * A boolean flag to indicate whether or not the xact run on this process
```
Comment (ctring) [inline src/include/storage/proc.h:311]: tag here

**Review by ctring — APPROVED** [review]

File: src/backend/storage/lmgr/proc.c (hunk 1 above)
Comment (poojanilangekar) [reply src/backend/storage/lmgr/proc.c:442]: Done.

File: src/include/storage/proc.h (hunk 2 above)
Comment (poojanilangekar) [reply src/include/storage/proc.h:312]: Done.

**Pull request merged**